src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.Api/Controllers/DataController.cs

```diff
@@ -150,6 +150,36 @@ public class DataController : ControllerBase
             }
         }
 
+        /// <summary>
+        /// Create a new data object of the defined data type
+        /// </summary>
+        /// <param name="dataType">The data type id</param>
+        /// <returns>Return a new instance of the data object including prefill and initial calculations</returns>
+        [Authorize]
+        [HttpPost]
+        [DisableFormValueModelBinding]
+        [RequestSizeLimit(REQUEST_SIZE_LIMIT)]
+        [ProducesResponseType(typeof(DataElement), 200)]
+        [Route("{org}/{app}/v1/data")]
```

Comment:
Consider moving this to the class to avoid duplication on each action.

Reply:
Hm, I noticed now that it's the same controller for both instance related data an non-instance related data. I think it would be clearer to keep those as two separate controllers as the context is different and the we keep the instance based controllers to instance related actions.

Reply:
Yeah, I considered that as well.. But what would we call such a controller? StatelessDataController okay?

Reply:
Yes, why not - I think that states the intent. Could consider NoInstanceDataController perhaps although I think your suggestions has a better "flow" to it:)